src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.Android.cs

```diff
@@ -13,7 +13,7 @@
 using System.Drawing;
 using Uno.UI;
 using Microsoft.Extensions.Logging;
-using static Uno.UI.MathEx;
+using static Uno.Extensions.MathEx;
```

Comment:
That's required?

Reply:
Yes, I moved this helper to Uno (was in Uno.UI). It's internal, so it's not a API breaking change